app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -57,8 +47,8 @@
  * @author ctsims
  *
  */
-@ManagedUi(R.layout.first_start_screen)
-public class CommCareSetupActivity extends CommCareActivity<CommCareSetupActivity> implements ResourceEngineListener{
+@ManagedUi(R.layout.first_start_screen_modern)
+public class CommCareSetupActivity extends CommCareActivity<CommCareSetupActivity> implements ResourceEngineListener, SetupEnterURLFragment.URLInstaller {

```

Comment:
This refactor is awesome! I was just trying to clean up this file myself and you breaking things up into fragments makes it so much better!